docs/conf.py

```diff
@@ -15,7 +15,46 @@
 
 # -- Project information -----------------------------------------------------
 
+import os
+import sys
+import tempfile
+import importlib
+import subprocess
+
+sys.path.append(os.path.abspath('..'))
+
 import honeycomb
+import services
+import integrations
+
+from honeycomb.servicemanager import base_service
+from honeycomb.integrationmanager import integration_utils
+
+services_list = next(os.walk('../services'))[1]
```

Comment:
so we're definitely not doing "" anymore?

Reply:
I didn't actually think about it at all..  most of this file uses `'` so I'll stick to that